Patch to point to the correct bin directory in the EASYBUILDINSTALLDIR and to
to the correct location of the ZLIB libraries
Thekla Loizou July 2015
--- bsmap-2.74/makefile.orig	2012-09-20 23:01:57.000000000 +0300
+++ bsmap-2.74/makefile	2014-12-16 09:24:16.033631393 +0200
@@ -1,6 +1,6 @@
 CC=	g++
 
-BIN = $(DESTDIR)/usr/bin
+BIN = $(DESTDIR)/bin
 FLAGS= -DMAXHITS=1000 -DTHREAD -funroll-loops -Lsamtools -Isamtools -Lgzstream -Igzstream -O3 -m64
 #FLAGS= -DMAXHITS=1000 -funroll-loops -Lsamtools -Isamtools -Lgzstream -Igzstream -O3 -Wall -Wno-strict-aliasing -m64
 
@@ -16,7 +16,7 @@
 bsmap: $(OBJS1)
 	(cd samtools; make)
 	(cd gzstream; make)
-	$(CC) $(FLAGS) $^ -o $@ $(THREAD) -lbam -lz -lgzstream
+	$(CC) $(FLAGS) $^ -o $@ $(THREAD) -lbam -L$(EBROOTZLIB)/lib -lz -lgzstream
 	rm -f *.o
 
 clean:
